packages/carbon-addons-boomerang-react/src/components/FeatureSideNav/_featureSideNav.scss

```diff
@@ -0,0 +1,40 @@
+@include exports('addons-bmrg-feature-sidenav') {
+  .#{$prefix}--bmrg-feature-sidenav-container {
+    background-color: $ui-01;
+    display: flex;
+    flex-direction: column;
+    height: calc(100vh - 3rem);
+    width: 21.875rem;
+    &.--left-border {
+      border-right: 0.0625rem solid $active-ui;
+    }
+    &.--small {
+      width: 18.75rem;
+    }
+    .bx--accordion {
+      height: 100%;
+      overflow-y: auto;
+    }
+    .bx--accordion__content {
+      padding: 0 !important;
+    }
+    .bx--accordion__item {
+      list-style-type: none;
+      .bx--accordion__arrow {
+        position: absolute;
+        right: 1.5rem;
+      }
+      .bx--accordion__title {
+        color: var(--cds-text-01);
+        font-size: 0.875rem;
+        letter-spacing: 0.01rem;
+        line-height: 1.25rem;
+        text-transform: capitalize;
+      }
+    }
+  }
+  .#{$prefix}--bmrg-feature-sidenav-links {
+    height: 100%;
```

Comment:
In light of the other possible changes maybe we shouldn't set the height to the 100% and then have it flex-grow: 1 to take up the remaining space available?